gg-pac-team3-back/src/controllers/courseController.mjs

```diff
@@ -1,16 +1,36 @@
-require('dotenv').config();
-const admin = require('../config/firebase');
+import dotenv from 'dotenv';
+import admin from '../config/firebase.mjs';
+import axios from 'axios';
+import CourseModel from '../models/CourseModel.mjs';
+
+dotenv.config();
 const db = admin.firestore();
-const axios = require('axios');
-const CourseModel = require('../models/CourseModel'); 
+const COURSES = 'Courses';
+const USERS = 'users';
```

Comment:
Why is `'Courses'` capitalized, but `'users'` lowercase? Pick one and stick with it (I recommend lowercase, as that's what I see in the firestore documentation)